lib/wsdl.js

```diff
@@ -855,7 +862,9 @@ ElementElement.prototype.description = function(definitions, xmlns) {
   if (type) {
     type = splitQName(type);
     var typeName = type.name,
-      ns = xmlns && xmlns[type.prefix] || definitions.xmlns[type.prefix],
+      ns = xmlns && xmlns[type.prefix] ||
+        (definitions.xmlns[type.prefix] !== undefined && this.schemaXmlns[type.prefix]) ||
```

Comment:
when checking for `!== undefined` explicitly, we are aware that `null` would pass - is this intentional?

Reply:
Anyway, I could change `definitions.xmlns[type.prefix] !== undefined` into `definitions.xmlns[type.prefix]`

Reply:
>  I checked the presence or absence of the property

good catch, I didn' t think of that... 👍